exonum-java-binding-core/rust/src/storage/time_schema.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use exonum::storage::{Fork, Snapshot};
+use exonum_time::schema::TimeSchema;
+use jni::{objects::JClass, JNIEnv};
+use std::panic;
+use storage::db::{View, ViewRef};
+use utils::{self, Handle};
+
+enum SchemaType {
+    SnapshotSchema(TimeSchema<&'static Snapshot>),
+    ForkSchema(TimeSchema<&'static mut Fork>),
+}
+
+/// Returns pointer to created TimeSchemaProxy object
+#[no_mangle]
+pub extern "system" fn Java_com_exonum_binding_time_TimeSchemaProxy_nativeCreate(
```

Comment:
Why is this needed at all if there is no object providing _any_ useful operations?